examples/interoperability/app.js

```diff
@@ -0,0 +1,125 @@
+import 'babel-polyfill';
+/* global document, window,*/
+/* eslint-disable no-console */
+import React, {PureComponent} from 'react';
+import {render} from 'react-dom';
+import DeckGL, {
+  ScatterplotLayer, OrthographicViewport, COORDINATE_SYSTEM
+} from '../../src';
```

Comment:
In other expamples, we import `from 'deck.gl'` and use the shared webpack config to optionally "alias" that to `src`.